core/src/main/scala/sbt/librarymanagement/LibraryManagementSyntax.scala

```diff
@@ -0,0 +1,24 @@
+package sbt.librarymanagement
+
+abstract class LibraryManagementSyntax extends DependencyBuilders {
```

Comment:
Is it really worth it? The trait-to-interface is still somewhat fragile (e.g. `val` must be final without return types), and I don't know who would ever need to mix this in.